apps/codelab/src/app/directives/nextSlide.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, HostListener } from '@angular/core';
+import { SlidesDeckComponent } from '@codelab/slides/src/lib/deck/deck.component';
+
+@Directive({
+  selector: 'button[nextSlide]'
```

Comment:
I don't think we should be restricting this directive to a button tag, there are many other use cases/ways to make it accessible

Reply:
Actually I particaly agree with that but any way I removed that restriction.